frontend/pages/policies/PolicyPage/_styles.scss

```diff
@@ -62,16 +62,17 @@
   }
 
   &__back-link {
+    display: flex;
+    align-items: center;
     font-size: $x-small;
     color: $core-vibrant-blue;
     font-weight: $bold;
     text-decoration: none;
-
+    height: 20px;
```

Comment:
Adding a height to all instances is the main fix. Each browser was automatically setting the height of the element a little differently, which was causing the inconsistency.